package.json

```diff
@@ -12,6 +12,7 @@
     "eslint-plugin-graphql": "^3.1.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-react-hooks": "^2.1.2",
+    "fetch-mock": "^8.0.0",
```

Comment:
All network interaction should happen though the graphql API. If you are using fetch directly, it's bypassing our network abstraction.

Reply:
Okay, I will have to figure out graphql on Monday. Thank you.